apps/devhub/tasks.py

```diff
@@ -36,14 +37,14 @@
 log = logging.getLogger('z.devhub.task')
 
 
-def validate(file_, listed=None):
+def validate(file_, listed=None, callback=None):
     """Run the validator on the given File or FileUpload object, and annotate
     the results using the ValidationAnnotator. If a task has already begun
     for this file, instead return an AsyncResult object for that task."""
 
     # Import loop.
     from .utils import ValidationAnnotator
-    annotator = ValidationAnnotator(file_, listed=listed)
+    annotator = ValidationAnnotator(file_, listed=listed, callback=callback)
 
     task_id = cache.get(annotator.cache_key)
```

Comment:
I don't think so. If it is fetched from cache then it doesn't queue another one so no more tasks should be spawned.